back/engines/commercial/id_nemlog_in/app/lib/id_nemlog_in/nemlog_in_verification.rb

```diff
@@ -8,6 +8,16 @@ def verification_method_type
       :omniauth
     end
 
+    def entitled?(auth)
+      minimum_age = config[:minimum_age]
+      return true unless minimum_age
```

Comment:
I'm not 100% sure, but I think `config[:minimum_age]` can be a blank string in some cases.
```suggestion
      return true if minimum_age.blank?
```